src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -48,21 +49,21 @@ public override void OnEventWritten(string name, object payload)
                 var context = payload as HttpContext;
                 if (context == null)
                 {
-                    AspNetCoreInstrumentationEventSource.Log.NullPayload(nameof(HttpInMetricsListener), nameof(this.OnEventWritten));
+                    AspNetCoreInstrumentationEventSource.Log.NullPayload(nameof(HttpInMetricsListener), EventName, HttpServerDurationMetricName);
                     return;
                 }
 
                 try
                 {
                     if (this.options.Filter?.Invoke(HttpServerDurationMetricName, context) == false)
                     {
-                        AspNetCoreInstrumentationEventSource.Log.RequestIsFilteredOut(HttpServerDurationMetricName);
+                        AspNetCoreInstrumentationEventSource.Log.RequestIsFilteredOut(nameof(HttpInMetricsListener), EventName, HttpServerDurationMetricName);
```

Comment:
I don't think there is much value in logging EventName. It's always going to be ActivityStop for Metrics. We could probably use different EventSource events for Metrics.

Reply:
wouldn't it be helpful for other metric? where we have to call filter for both start and stop?

Reply:
My vote would be to merge this as-is because it is better than what we have and then @vishweshbankwar can continue to improve it as we work through the new metrics being added 😄